tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -33,6 +40,43 @@ Verify Content In RHODS Explore Section
     Check Number Of Displayed Cards Is Correct  expected_data=${EXP_DATA_DICT}
     Check Cards Details Are Correct   expected_data=${EXP_DATA_DICT}
 
+Verify Disabled Cards Can Be Removed
+    [Documentation]     Verifies it is possible to remove a disabled card from Enabled page.
+    ...                 It uses RHOSAK as example to test the feature
+    [Tags]    Sanity
+    ...       ODS-1081    ODS-1092
+    Enable RHOSAK
+    OpenShiftCLI.Delete    kind=ConfigMap    name=rhosak-validation-result    namespace=redhat-ods-applications
```

Comment:
I think lines 49-53 could be replaced by the existing keyword `Remove RHOSAK From Dashboard`

Reply:
yes you're right. Maybe when I started working on this TC that keyword wasn't merged yet. I'll replace, thanks!

Reply:
done